cirq-core/cirq/neutral_atoms/neutral_atom_devices.py

```diff
@@ -215,11 +223,12 @@ def validate_moment(self, moment: ops.Moment):
             assert isinstance(op, (ops.GateOperation, ops.ParallelGateOperation))
             for k, v in CATEGORIES.items():
                 assert isinstance(v, tuple)
-                if isinstance(op.gate, v):
+                op_gate = _subgate_if_parallel_gate(op.gate)
```

Comment:
nit: here, too, I'd suggest `gate` instead of `op_gate`.

Reply:
done